### PR TITLE
Add Script and Run Game

### PR DESCRIPTION
**Pull request by DeKaylaPhillips**
# Add Script and Run Game

## Description
This branch displays a welcome prompt, an empty board, and prompts a user to place a token on the board. This branch also removes the integration test suite (_suggestion by primary mentor, not included within corresponding ticket*_). 

### Type of Change
- New feature (non-breaking change that adds functionality)

### Tests
- _Manual testing in runner file_ (ticTacToe.js)

### CODE REVIEW

File: test/GameLogic/gameLogic.test.js [hunk 1]
```diff
@@ -76,5 +76,17 @@ describe('GameLogic', () => {
             const winDetected = game.containsWinningColumn();
             expect(winDetected).toEqual(false);
         });
+
+        test('will return false boolean value when a column filled with the same token is not detected', () => {
```
Comment (Naomi-Dennis) [inline test/GameLogic/gameLogic.test.js:80]: can you rename this test to "will return false boolean when a column is not filled with the same token"?

File: test/GameLogic/gameLogic.test.js [hunk 2]
```diff
@@ -99,7 +99,8 @@ describe('getWinningToken()', () => {
             const winDetected = game.getWinningToken() != null;
             expect(winDetected).toBe(true);
         });
-
+    
+
```
Comment (Naomi-Dennis) [inline test/GameLogic/gameLogic.test.js:103]: np: whitespace

**Review by Naomi-Dennis — APPROVED** [review]

File: test/GameLogic/gameLogic.test.js (hunk 2 above)
Comment (DeKaylaPhillips) [reply test/GameLogic/gameLogic.test.js]: Whitespace removed. :)

File: lib/TicTacToe/ticTacToe.js [hunk 3]
```diff
@@ -1,21 +1,35 @@
 import { Board } from '../Board/board.js'
 import { Display } from '../Display/display.js'
 import { UserPrompts } from '../UserPrompts/userPrompts.js';
+import { GameLogic } from '../GameLogic/gameLogic.js';
+import { Token } from '../Token/token.js';
+
```
[the remark this answers is not in the document]
Comment (DeKaylaPhillips) [reply lib/TicTacToe/ticTacToe.js]: Whitespace removed. :)

**Pull request merged**